charts/cortx/templates/_cluster.yaml.tpl

```
@@ -1,7 +1,13 @@
+### TODO Revisit UUID defaults here since we are moving away from UUID entirely...
 {{- define "storageset.node" -}}
 - name: {{ .name }}
+  {{- if eq .type "server_node" }}
+  id: {{ default uuidv4 .id | quote }}
+  hostname: {{ .hostname }}
+  {{- else }}
   id: {{ default uuidv4 .id | replace "-" "" | quote }}
   hostname: {{ .name }}
+  {{- end }}
```

```suggestion
  {{- else }}
  id: {{ default uuidv4 .id | replace "-" "" | quote }}
  {{- end }}
  hostname: {{ coalesce .hostname .name }}
```
Not a big deal, but slightly simpler.